packages/frontend/apps/electron/test/main/recording-state.spec.ts

```
@@ -9,108 +10,475 @@ vi.mock('../../src/main/logger', () => ({
+        isMacOS: () => false,
+        isWindows: () => false,
+        resolveExistingPathInBase,
+      };
+    });
+
+    vi.doMock('../../src/main/shared-storage/storage', () => ({
+      globalStateStorage: {
+        get: (key: string) => storageState.get(key),
+        set: (key: string, value: unknown) => {
+          storageState.set(key, value);
+          const subject$ = watchSubjects.get(key);
```

ESLint

[error] Finnish notation should not be used here.

(rxjs/finnish)